docker.go

```diff
@@ -142,24 +193,37 @@ func (docker *Docker) ExposePort(containerPort, hostPort, servicePort int, proto
 //		value:		the value of the option
 func (docker *Docker) Parameter(key string, value string) *Docker {
```

Comment:
done ... 
Additionally, I renamed the method from `Parameter` -> `AddParameter` to be more in line with other, similar methods. I can revert this though.